client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -118,7 +139,10 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
             .get('pythonSettings')
             .get('djangoSettingsModule')
             .updateValueAndValidity();
-        } else if (this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python && val === PythonFrameworkType.Flask) {
+        } else if (
+          this.wizard.wizardValues.buildSettings.applicationFramework === WebAppFramework.Python &&
+          val === PythonFrameworkType.Flask
+        ) {
```

Comment:
I think this is just breaking into multiple lines. Avoid it if that is the case

Reply:
Done